docgen/src/guides/v3-migration.md

```diff
@@ -121,8 +122,8 @@ With the redo button:
 | `escapeHits`    | `escapeHTML`    |
 | `showMoreLabel` | `loadMoreLabel` |
 
-- `escapeHTML` defaults to `true`
-- `allItems` template has been removed in favor of `connectHits`
+* `escapeHTML` defaults to `true`
+* `allItems` template has been removed in favor of `connectHits`
```

Comment:
these changes probably weren't on purpose in this PR

Reply:
It doesn't really matter because we override the migration guide before each merge.